src/Settings/SettingsTab.php

```diff
@@ -29,16 +29,10 @@ function ( $links ) {
 		);
 
 		add_filter(
-			'woocommerce_get_settings_advanced',
-			function( $settings, $current_section ) {
-				if ( 'features' !== $current_section ) {
-					return $settings;
-				}
-
-				return $this->add_experimental_settings( $settings );
-			},
-			100,
-			2
+			'woocommerce_settings_features',
```

Comment:
I couldn't find this [hook in code ref](https://woocommerce.github.io/code-reference/hooks/hooks.html) so I figured this could be a dynamically created hook. I was able to find the [source of this hook](https://github.com/woocommerce/woocommerce/blob/e5b26f0b7d78f137c8ab76f79c371681c59573f5/plugins/woocommerce/includes/admin/views/html-admin-settings.php#L40). But I couldn't find the trace for `woocommerce_get_settings_advanced` hook. Can you point me in the right direction?

Reply:
Thank you, Bartosz. Resolving this.